.github/CODEOWNERS

```diff
@@ -120,6 +120,7 @@
 
 # mlir-llvm
 /mlir/**/LLVM/ @llvm/pr-subscribers-mlir-llvm
```

Comment:
Shall we just remove the trailing `/` here?

Reply:
Just tested this using `git check-ignore` (since this should be the same syntax as for the codeowners file apparently) and that at least did not work for any checks in the `LLVMIR` directory. 
Changing it to either `LLVMIR` with or without the slash works, but it seems to me like it doesn't treat trailing positions specially.

Unless GitHub CodeOwners files do it differently.

Reply:
OK! We need a trailing `*`, which means most of the existing patterns are incorrect... 
`mlir/**/LLVM*` should match both here.